account_banking_mandate/models/account_move_line.py

```diff
@@ -16,14 +17,21 @@ class AccountMoveLine(models.Model):
     def _prepare_payment_line_vals(self, payment_order):
         vals = super(AccountMoveLine, self)._prepare_payment_line_vals(
             payment_order)
-        if payment_order.payment_type == 'inbound' and self.mandate_id:
-            vals['mandate_id'] = self.mandate_id.id
-            vals['partner_bank_id'] = self.mandate_id.partner_bank_id.id
-        partner_bank_id = vals.get('partner_bank_id', False)
-        if partner_bank_id and 'mandate_id' not in vals:
-            mandate = self.env['account.banking.mandate'].search(
-                [('partner_bank_id', '=', partner_bank_id),
-                 ('state', '=', 'valid')], limit=1)
-            if mandate:
-                vals['mandate_id'] = mandate.id
+        if payment_order.payment_type != 'inbound':
```

Comment:
bank account is not set on outbound operations?

Reply:
Of course, but not in mandate module...
https://github.com/OCA/bank-payment/blob/764f22fa14e88c3e82474e92760d6a993c099035/account_payment_order/models/account_move_line.py#L51-L56

Reply:
OK, thanks